### ISSUE TITLE
Elementor Editor translation regardless of user language

### ISSUE THREAD

**Issue by madaloth** [elementor/elementor#16373]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

My language is a right-to-left language (Hebrew), so I choose Hebrew as the language of the website, but to work in the admin environment, I usually prefer English as the language of my admin environment, which is a  left-to-right language, I do this by changing the user language in the user settings to English.
The problem is that the Elementor editor environment, regardless of the user language, is always the language of the website, which does not make sense, and other plugins usually select the admin language based on the user language.

## Steps to reproduce

easy!
1. install WordPress and Elementor.
2. change the website language to Hebrew.
3. change the admin user language to English.
4. edit a page with Elementor.
5. the elementor editor is in Hebrew.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by lllopo** [comment]
Yup, this was fixed and in the lates releases - 3.7 etc is broken again.
